packages/e2e-tests/src/specs/editor/prepublishChecklist.js

```diff
@@ -51,4 +53,29 @@ describe('Pre-Publish Checklist', () => {
     await expect(page).toMatch('Add poster image');
     await percySnapshot(page, 'Prepublish checklist');
   });
+
+  describe('Contributor User', () => {
+    withUser('contributor', 'password');
+
+    it('should not let me publish a story as a contributor', async () => {
+      await createNewStory();
+
+      await insertStoryTitle('Publishing Flow: Contributor');
+
+      const checklistToggle = await page.$(
+        'button[aria-owns="checklist_companion"]',
+        { text: 'Checklist' }
+      );
+
+      // verify no issues are present
+      await checklistToggle.click();
```

Comment:
Could also just do it like the test above:

```js
await expect(page).toClick('button[aria-label^="Checklist: "]');
await expect(page).toMatchElement(
  '#pre-publish-checklist[data-isexpanded="true"]'
);
```

Any reason why this test would need to do it differently?

Reply:
Nope that's a good catch. Just moved this over, so I can update that here